charts/akeyless-k8s-secrets-injection/values.yaml

```diff
@@ -4,6 +4,10 @@
 
 deployment:
   labels: {}
+  # Remove the {} and add any needed values to your SecurityContext
```

Comment:
why do i see SecurityContext twice?

Reply:
one is on the deployment level and the other is on the containers level